spec/factories/course_survey_questions.rb

```diff
@@ -1,13 +1,48 @@
 # frozen_string_literal: true
 FactoryGirl.define do
   sequence(:course_survey_question_name) { |n| "Survey Question #{n}" }
-  factory :course_survey_question, class: Course::Survey::Question.name do
+  factory :course_survey_question, class: Course::Survey::Question.name, aliases: [:question] do
```

Comment:
I think we can use `survey_question` as the alias instead of `question` - might mix up with assessment.